pylint/checkers/typecheck.py

```diff
@@ -182,6 +183,13 @@ def _missing_member_hint(owner, attrname, distance_threshold, max_choices):
               'no-member',
               'Used when a variable is accessed for an unexistent member.',
               {'old_names': [('E1103', 'maybe-no-member')]}),
+    'I1101': ('%s %r has not %r member, but source is unavailable. Consider '
+              'adding this package to extension-package-whitelist if you want '
+              'to perform analysis based on run-time introspection of package.',
+              'c-extension-no-member',
```

Comment:
Maybe even `c-extension-encountered` or something like that? the fact that we catch this case in `no-member` check seems to be quite irrelevant.